common/simulation.hpp

```diff
@@ -47,15 +47,17 @@ namespace tec {
 		using EventQueue<ClientCommandsEvent>::On;
 		using EventQueue<ControllerAddedEvent>::On;
 		using EventQueue<ControllerRemovedEvent>::On;
-		void On(std::shared_ptr<KeyboardEvent> data) override;
-		void On(std::shared_ptr<MouseBtnEvent> data) override;
-		void On(std::shared_ptr<MouseMoveEvent> data) override;
-		void On(std::shared_ptr<MouseClickEvent> data) override;
-		void On(std::shared_ptr<ClientCommandsEvent> data) override;
-		void On(std::shared_ptr<ControllerAddedEvent> data) override;
-		void On(std::shared_ptr<ControllerRemovedEvent> data) override;
-		void On(std::shared_ptr<FocusCapturedEvent> data) override;
-		void On(std::shared_ptr<FocusBlurEvent> data) override;
+		using EventQueue<FocusCapturedEvent>::On;
```

Comment:
The real fix was by adding the explicit using instead of adding the override keyword.